portfolio/src/main/webapp/pom.xml

```diff
@@ -0,0 +1,44 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
This file shouldn't be needed.  You can run maven from the portfolio directory

Reply:
Ok I'll delete it from this directory, i think it got copied here by accident